packages/reactivity/src/computed.ts

```diff
@@ -68,10 +68,47 @@ export class ComputedRefImpl<T> {
   }
 }
 
+/**
+ * Creates an immutable reactive ref object.
+ *
+ * The `getter` function's return value determines the ref's value.
+ *
+ * @example
+ * ```js
+ * const count = ref(1)
+ * const plusOne = computed(() => count.value + 1)
+ *
+ * console.log(plusOne.value) // 2
+ * plusOne.value++ // error
+ * ```
+ *
+ * @param getter Function that produces the next value.
```

Comment:
Hmm not so sure about this. I _deliberately_ omitted the `debugOptions` parameters since they're not mentioned on the documentation website, neither.

Depends on whom these comments are for: Users building stuff with Vue - and therefore relying on the "public" API of those methods? Or users building Vue itself - and relying on the "internals" of the API, as well?

Reply:
It's actually there: https://vuejs.org/api/reactivity-core.html#computed. It's for debugging purposes only but also accessible to Vue users

Reply:
I've added the `param` descriptions plus a link to https://vuejs.org/guide/extras/reactivity-in-depth.html#computed-debugging. However, `@internal` doesn't really work on single parameters only.